docs/UserGuide.md

```diff
@@ -426,45 +393,54 @@ Invalid Input Example | Application Output
 **optout notifications** | Invalid `NOTIFICATION_DESCRIPTION` (refer to aforementioned for the list of `NOTIFICATION_DESCRIPTION` to enter).
 **optout** | `NOTIFICATION_DESCRIPTION` cannot be empty.
 
-### Notes feature
-![Window with Notes](images/notes/window_with_notes.png)
-![Notes Window](images/notes/notes_window.png)
+### Notes
 
-#### 1. Adding Notes to a Person
+<div style="display:flex; justify-content:space-around; align-items:center;">
+  <img src="images/notes/window_with_notes.png" alt="Window with Notes" style="height:400px; margin:10px;">
+  <img src="images/notes/notes_window.png" alt="Notes Window" style="height:400px; margin:10px;">
+</div>
 
-##### Command Format:
-    note INDEX NOTE_CONTENT
+Allows you to add notes to a person and remove notes from a person.
 
-##### Parameters:
-- `INDEX`: The position of the person in the list you want to add a note to. This should be a positive integer.
-- `NOTE_CONTENT`: The content of the note you want to add.
+You can add notes to a person with the `addnote` command, remove notes from them with the `removenotes` command.
 
-##### Example:
-If you want to add a note to the person at position 1 in the list, you would use:
-    
-    note 1 This is a sample note for the person.
+You can view notes by one of two ways: by using the `addnote` command, or by clicking on the `Notes` button in the person's information window.
 
-This will add a note "This is a sample note for the person." to the person at index 1.
+Format: `addnote PERSON_INDEX NOTE_CONTENT`, `removenote PERSON_INDEX NOTE_INDEX` and `notes PERSON_INDEX`
 
-#### 2. Removing Notes from a Person
+<box type="warning">
+
+Always make sure the indices provided are valid and within the bounds of the list. Invalid indices will result in an error.
+
+</box>
+<box type="info">
+
+The fields you enter should follow the following format:
 
-##### Command Format:
-    removenote INDEX_PERSON INDEX_NOTE
+| Parameter     | Description                                                                                                 | 
+|---------------|-------------------------------------------------------------------------------------------------------------| 
+| `PERSON_INDEX`| The position of the person in the list you want to add a note to. This should be a positive integer, and should be within the bounds of the list. | 
+| `NOTE_INDEX`  | The position of the note in the person's list of notes you want to remove. This should be a positive integer, and should be within the bounds of the list. | 
+| `NOTE_CONTENT`| The content of the note you want to add. It has to be non-empty, and can contain any character.              | 
 
-##### Parameters:
-- `INDEX_PERSON`: The position of the person in the list you want to remove a note from. This should be a positive integer.
-- `INDEX_NOTE`: The position of the note in the person's list of notes you want to remove. This should be a positive integer.
+</box>
 
-##### Example:
+<box>
```

Comment:
Both boxes should be combined and changed to an `info`, would be neater if it could be formatted into a table, or with some neat separator added between the two examples